stock_reserve_rule/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1 @@
+* Guewen Baconnier <guewen.baconnier@camptocamp.com>
```

Comment:
you?

Reply:
Don't know, didn't do so much to migrate it :)